zulipterminal/ui_tools/boxes.py

```diff
@@ -314,6 +314,10 @@ def _set_stream_write_box_style(self, widget: ReadlineEdit, new_text: str) -> No
             (color, stream_marker)
         )
 
+    def update_topic_compose_header(self, new_topic: str) -> None:
+        self.title_write_box.set_edit_text(new_topic)
+        self.title_write_box.set_edit_pos(len(new_topic))
```

Comment:
Did you want to assert on positions too, generally?

Reply:
Sounds good, I'll update accordingly.